<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>제작 물품</string>
      </property>
      <widget class="QListWidget" name="listWidget_products">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>261</width>
         <height>131</height>
        </rect>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_production_team">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>261</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>철무방</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>만금당</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>성군당</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>약왕원</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>일미문</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>태상문</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>수익</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;판매수익&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_revenue">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>설정</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>271</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>수수료</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>131</width>
             <height>151</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>판매가 100금 이상: 6%</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>1000금 이상: 8%</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>10000금 이상: 10% 권장</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cb_charge">
              <item>
               <property name="text">
                <string>5%</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6%</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7%</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8%</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>9%</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10%</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>11%</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>판매 가격</string>
          </property>
          <widget class="QRadioButton" name="rb_lowest">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>90</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>시장 최저가</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QRadioButton" name="rb_direct">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>90</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>직접 입력</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_price">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_product_count">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>111</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>맑은 고딕</family>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_all_price">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>111</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>맑은 고딕</family>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>제작 비용</string>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>279</width>
         <height>111</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>필요 개수</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>현재 가격</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>총 가격</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>영석</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>월석</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>영단</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>선단</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="lbl_production_cost">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>제작 수수료:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_cost">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>총 제작비:</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
